05_testing_and_ci/intro_slides.md

```diff
@@ -29,16 +29,9 @@ slideOptions:
 
 ## Learning goals of the chapter
 
-- What is it important to test simulation software
-- What are the concepts of unit testing, integration testing and regression testing and how to write such tests for a simulation software
-- How to use the Python testing frameworks `pytest` and `unittest` to generate a automated testing workflow
-
----
-
-## Lets start with a survey
-
-- https://app.sli.do/event/9xfogBPKmJxnScNrM4nUUZ
-- **ID: 407683**
+- Learn the importance of test simulation software
+- Discern the concepts of unit testing, integration testing and regression testing with the perspective of simulation software
+- Getting familiar with the Python testing frameworks `pytest` and `unittest`
```

Comment:
When writing learning goals, always think: "After studying this module or lecture (including the exercise), students should be able to ..."

With this, item 1 and 3 do not work.